src/Opdex.Platform.Common/Enums/TransactionEventType.cs

```diff
@@ -16,8 +16,8 @@ public enum TransactionEventType
     // Liquidity pool logs
     AddLiquidityEvent = 8,
     RemoveLiquidityEvent = 9,
-    SwapEvent = 10,
-    ReservesChangeEvent = 11,
+    ReservesChangeEvent = 10,
+    SwapEvent = 11,
     StartStakingEvent = 12,
```

Comment:
Not sure about this, will need to check out mappings. Was there a need for this change? 

Our TransactionLogTypes and database have Swap at 10 and Reserves at 11.

Reply:
My bad, I changed this to match the order they were listed in OpenAPI. Should have updated the order in OpenAPI, if anything. Reverted